cosmwasm/contracts/crosschain-registry/src/execute.rs

```diff
@@ -414,7 +445,14 @@ pub fn chain_to_prefix_operations(
                 CHAIN_TO_BECH32_PREFIX_MAP.save(
                     deps.storage,
                     &chain_name,
-                    &(map_entry.value, true).into(),
+                    &(map_entry.value.clone(), true).into(),
+                )?;
+                // Add to the reverse map of the enabled prefix
+                push_to_map_value(
+                    deps.storage,
+                    &CHAIN_TO_BECH32_PREFIX_REVERSE_MAP,
+                    &map_entry.value,
+                    chain_name.clone(),
```

Comment:
I don't think this is correct for the enable / disable logic right? We need to set the boolean entry of the tuple to true here

Reply:
I have no concept of enable/disable on the reverse map. It's basically a representation of the current state, so when we enable the value, we "add it" to the reverse map; when we disable it, we remove it. Does that make sense?

Reply:
Makes sense, thank you!